src/main/java/com/coolkids/coolKidsApp/controllers/EventController.java

```diff
@@ -11,31 +13,16 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
+@RestController
 @RequestMapping(path = "api/v1/")
 @AllArgsConstructor
 public class EventController {
+    private final EventService eventService;
 
-    //Todo: list all events
-
-
-
-    //Todo: create event
-
-
-    //Todo: delete event
-
-
-    //Todo: get an event by id
```

Comment:
I think the todos are still missing

Reply:
See if you see them in the latest commit.

Reply:
I still don't see them in the latest changes. Maybe try running a git add and then recommit

Reply:
See my last commit. GitHub likes to minimize their lines.